internal/go/skycfg/proto_api.go

```diff
@@ -25,8 +25,9 @@ import (
 
 	"github.com/golang/protobuf/jsonpb"
 	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
 	"go.starlark.net/starlark"
-	yaml "gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v2"
```

Comment:
Can you keep this import alias as-is? It's nice to have explicitly named imports when upstream uses a non-default package name.

Reply:
I think this is a holdover/cargo-cult from the first iteration of this change. Let me change the alias back.